cqrs-user-command-service/src/main/java/dev/amir/usercommand/application/usecase/UserUseCases.java

```diff
@@ -17,31 +19,36 @@ public class UserUseCases implements UserInputPort {
 
     @Override
     public String createUser(User user) {
+        log.info("Verifying if the 'id' field is empty for user creation.");
         if (StringUtils.hasText(user.getId())) {
             throw new IllegalArgumentException("Invalid User, id field must be empty");
         }
 
         User savedUser = userOutputPort.save(user);
+        log.info("User successfully created and saved with id: " + savedUser.getId());
         userMessageOutputPort.sendMessage(savedUser);
         return savedUser.getId();
     }
 
     @Override
     public void updateUser(User user) {
+        log.info("Verifying if the 'id' field exists for user update.");
         if (!StringUtils.hasText(user.getId())) {
             throw new IllegalArgumentException("Invalid User, id field must exist");
         }
 
         User savedUser = userOutputPort.save(user);
+        log.info("User successfully updated with id: " + savedUser.getId());
         userMessageOutputPort.sendMessage(savedUser);
     }
 
     @Override
     public boolean deleteUser(String userId) {
+        log.info("Attempting to delete user with ID: " + userId);
         if (!StringUtils.hasText(userId)) {
             throw new IllegalArgumentException("Invalid User, id field must exist");
         }
-
+        log.info("User deletion request accepted ID:" + userId);
```

Comment:
in order to follow the same message format like line 41 and line 28. change this message to:
`"User successfully deleted with id"`

Reply:
resolved